### PR TITLE
Update/add monthly breakdown to composite checkout line items

### PR DESCRIPTION
**Pull request by nbloomf**
#### Changes proposed in this Pull Request

* Add a monthly breakdown to plan line items in composite checkout. Depends on D44051-code

To do:
- [x] Needs design fixup

#### Testing instructions

Enter checkout with a plan; the line item should include an (accurate!) monthly breakdown of the plan cost before coupons or discounts. The monthly price should match that shown on the plans page. Check that the monthly price updates when changing the plan length.

Screen Shot 2020-05-27 at 1 57 45 PM

### CODE REVIEW

**Comment by fditrapani** [comment]
I made some minor tweaks. Looks good now!

image

File: client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js [hunk 1]
```diff
@@ -73,6 +73,20 @@ function WPLineItem( {
 					{ item.wpcom_meta?.is_bundled && item.amount.value === 0 && (
 						<DiscountCalloutUI>{ translate( 'First year free' ) }</DiscountCalloutUI>
 					) }
+					{ 'plan' === item.type && item.wpcom_meta?.months_per_bill_period && (
+						<LineItemMonthlyBreakdown
```
[the remark this answers is not in the document]
Comment (nbloomf) [reply client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js]: Good catch. The most recent commit extracts that logic to a variable - let me know if this is what you had in mind.

Screen Shot 2020-05-28 at 8 54 10 AM

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3696842

Thank you @nbloomf for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3696842).

**Pull request merged**